wp-content/themes/csisnuclearnetwork/assets/_scss/components/_communication-block-widget.scss

```diff
@@ -12,7 +12,7 @@
     @extend %text-style-label-medium;
   }
 
-  /* stylelint-disable-next-line */
+  /* stylelint-disable */
```

Comment:
This was already handled, so you'll need to do a git pull from dev and rebase on your branch.